### PR TITLE
Decouple from Templating Component, Stage 1

### PR DESCRIPTION
**Pull request by covex-nn**
I am targeting this branch, because this is a patch.

## Changelog

```markdown
### Changed
- Switch all templates references to Twig namespaced syntax
- Switch from templating service to sonata.templating
```

## To do

- [X] Switch all templates references to Twig namespaced syntax. Update docs, src and tests
- [x] Add `sonata-project/block-bundle` to composer.json
- [x] Switch from `templating` service to `sonata.templating`

## Subject

This PR is a part of a big plan of decoupling sonata-project bundles from Templating Component. See sonata-project/dev-kit#380 for details

### CODE REVIEW

File: src/Resources/views/Form/form_admin_fields.html.twig [hunk 1]
```diff
@@ -9,24 +9,24 @@ file that was distributed with this source code.
...
 {% endblock %}
 
 {% block sonata_admin_orm_many_to_one_widget %}
-    {% include 'SonataAdminBundle:CRUD/Association:edit_many_to_one.html.twig' %}
+    {% include '@SonataAdmin/CRUD/Association/edit_many_to_one.html.twig' %}
 {% endblock %}
 
 {% block sonata_admin_orm_one_to_many_widget %}
-    {% include 'SonataAdminBundle:CRUD/Association:edit_one_to_many.html.twig' %}
+    {% include '@SonataAdminBundle/CRUD/Association/edit_one_to_many.html.twig' %}
```
Comment (jordisala1991) [inline src/Resources/views/Form/form_admin_fields.html.twig]: Bundle needs to be removed

Comment (covex-nn) [reply src/Resources/views/Form/form_admin_fields.html.twig]: Fixed

**Comment by covex-nn** [comment]
Please, restart failed travis jobs

**Comment by covex-nn** [comment]
@jordisala1991 please review

**Comment by greg0ire** [comment]
Thanks @covex-nn !

**Pull request merged**